examples/cpp/aot_inductor/resnet/src/resnet_handler.hh

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <folly/FileUtil.h>
+#include <folly/dynamic.h>
+#include <folly/json.h>
+#include <fmt/format.h>
```

Comment:
Better to import only header files inside the header which are actually necessary. Everything else should be imported inside the cpp file.